### PR TITLE
Feature/tf 89 automatically capture resolution during setup

### PR DESCRIPTION
**Pull request by MattGrayUL**
- [x] Code Review
- [x] Developer Testing: Key item tests
- [ ] Product Owner Review (Optional)

### CODE REVIEW

File: TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/Screens/QuickSetupCompleteScreen.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using UnityEngine;
+using Ultraleap.TouchFree.ServiceShared;
```
Comment (PeterNancollis-UH) [inline TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/Screens/QuickSetupCompleteScreen.cs]: This doesn't appear to be used, is it necessary?

File: TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/ManualConfigUI.cs [hunk 2]
```diff
@@ -43,6 +43,12 @@ private void Awake()
             resetToDefaultWarning.SetActive(false);
         }
 
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+            VirtualScreen.CaptureCurrentResolution();
```
Comment (PeterNancollis-UH) [inline TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/ManualConfigUI.cs:49]: In testing this I found that the captured resolution is applied to the visuals of the manual setup screen, but it isn't applied to the cursor position until you leave the manual setup screen. Is this the expected behaviour?

I found myself a little confused as the cursor was wrong while on the screen, then it was magically correct after I left.

File: TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/Screens/QuickSetupCompleteScreen.cs (hunk 1 above)
Comment (MattGrayUL) [reply TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/Screens/QuickSetupCompleteScreen.cs]: It isn't, I think this is carry-over from one of the other commits. Will delete

File: TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/ManualConfigUI.cs (hunk 2 above)
Comment (MattGrayUL) [reply TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/ManualConfigUI.cs:49]: This is interesting as I would not expect that to be the case. It changes the text file which the Service listens to and should immediately update the Interaction (and so, the cursor position)

Comment (PeterNancollis-UH) [reply TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/ManualConfigUI.cs:49]: The latest commit fixes this issue. The cursor now updates to the newly captured resolution when the screen loads

**Review by PeterNancollis-UH — APPROVED** [review]

**Pull request merged**